core-api/src/main/java/com/optimizely/ab/decision/experiment/CompositeExperimentDecisionService.java

```diff
@@ -0,0 +1,96 @@
+/****************************************************************************
```

Comment:
The name `Decision` in the classpath is redundant. We can just call it `CompositeExperimentService`. Need to rename all other classes as well.